source/blog/2015-12-21-understanding-docker-top-and-ps.html.md

```diff
@@ -0,0 +1,219 @@
+---
+title: atomic top, docker top and ps
+author: runcom
+date: 2015-12-21 17:11:47 UTC
+tags: Docker
+comments: true
+published: true
+---
+
+`atomic top` is a new `atomic` subcommand which will allow you to interactively display information about containers running on an atomic host. Under the hood `atomic top` is using the Docker's `top` API to gather information about a particular container. When queried, the Docker daemon uses `ps` to give back results. The same happens with the Docker CLI as well.  
```

Comment:
`atomic top` is a new `atomic` subcommand which will allow you to interactively display the `top` process information from `all` of your containers running on a host. It works like standard, top command, where it will continuously update the top processes. Under the hood `atomic top` is using the Docker's `top` API to gather information about a particular container. When queried, the Docker daemon uses `ps` to give back results. The same happens with the Docker CLI as well.